site/pages/disruption-detail/[disruptionId].page.tsx

```diff
@@ -602,11 +602,19 @@ const DisruptionDetail = ({
                         </div>
                         <Link
                             role="button"
-                            href={{
-                                pathname: `${TYPE_OF_CONSEQUENCE_PAGE_PATH}/${disruption.disruptionId}/${nextIndex}`,
-                                query: { return: DISRUPTION_DETAIL_PAGE_PATH },
-                            }}
-                            className="govuk-button mt-2 govuk-button--secondary"
+                            href={
```

Comment:
Might be nicer if we make it so the Link wraps a button here and then the button styles can be moved to that, then we can use the proper `disabled` and `aria-disabled="true" props

Reply:
done